services/sqs/src/test/java/software/amazon/awssdk/services/sqs/SqsBatchManagerTest.java

```diff
@@ -206,8 +216,8 @@ public void sendMessageBatchNetworkError_causesConnectionResetException() {
 
         CompletableFuture<SendMessageResponse> response1 = responses.get(0);
         CompletableFuture<SendMessageResponse> response2 = responses.get(1);
-        assertThatThrownBy(response1::join).isInstanceOf(CompletionException.class).hasMessageContaining("Connection reset");
-        assertThatThrownBy(response2::join).isInstanceOf(CompletionException.class).hasMessageContaining("Connection reset");
+        assertThatThrownBy(response1::join).isInstanceOf(CompletionException.class);
+        assertThatThrownBy(response2::join).isInstanceOf(CompletionException.class);
```

Comment:
Can we assert the cause of the `CompletionException`? `CompletionException` can be anything. Similar to `sendMessageBatchFunctionReturnsWithError_completeMessagesExceptionally`

Reply:
Asserting cause of `CompletionException` with hasMessageContaining (different one from the original since original one was causing a build failure).